lib/iruby/kernel.rb

```diff
@@ -33,8 +36,8 @@ def initialize(config_file)
           hb_socket = c.socket(ZMQ::REP)
           hb_socket.bind(connection % @config['hb_port'])
           ZMQ::Device.new(hb_socket, hb_socket)
-        rescue Exception => ex
-          STDERR.puts "Kernel heartbeat died: #{ex.message}\n"#{ex.backtrace.join("\n")}"
```

Comment:
I think we have already catch exception [in command.rb](https://github.com/SciRuby/iruby/pull/12/files#diff-9d5692ae3ea9c349b6d32a29d63fb1eeL62) so maybe it's unnecessary to catch them here.